code/parse/sexp.cpp

```diff
@@ -33799,8 +33799,8 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 
 	// wookieejedi
 	{ OP_GET_HOTKEY, "get-hotkey\r\n"
-		"\tReturns the hotkey integer of the ship or wing.\r\n"
-		"\tNo hotkey returns -1, F5 returns 0, F6 returns 1, ...\r\n\r\n"
+		"\tReturns the hotkey integer of the ship or wing. Return values range from -1 to 8.\r\n"
+		"\tNo hotkey returns -1, F5 returns 0, F6 returns 1, ..., F12 returns 7, and 'Hidden' hotkey returns 8.\r\n\r\n"
```

Comment:
I don't think the 'Hidden' hotkey is actually a value that can be returned in-mission.  It's only used during mission load.